apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue

```diff
@@ -18,12 +18,13 @@
                 </p-data-loader>
             </div>
             <div class="table-pagination-wrapper">
-                <p-text-pagination :this-page.sync="state.thisPage"
-                                   :disable-next-page="!state.showNextPage"
+                <p-text-pagination :this-page="state.thisPage"
+                                   :disable-next-page="!state.data?.more"
+                                   @update:thisPage="handleUpdateThisPage"
                 >
                     <template #default>
                         <span class="this-page">{{ state.thisPage }}</span>
-                        <span v-if="state.showNextPage"> / ...</span>
+                        <span v-if="state.data?.more"> / ...</span>
```

Comment:
issue: Please remove optional chaining from vue template because vue2 cannot parse this. :)

Reply:
Oh thanks!

Reply:
I didn't get any warnings.. it doesn't even work.